requirements/base.txt

```diff
@@ -2,6 +2,6 @@
 # Runtime requirements are handled in setup.py
 -e .
 ddt
-edx-lint
+git+https://github.com/edx/edx-lint.git@clintonb/python-3.5#egg=edx-lint==0.5.0
```

Comment:
This can be reverted once https://github.com/edx/edx-lint/pull/38 is merged. For now, this PR is blocked.